### PR TITLE
Removed Decision column from Active games table ...

### LINKED ISSUE #1829: New overview tables are too wide
The new overview tables are wider than they used to be, which makes them uglier horizontally on smaller (in my case, about 1200 pixels wide). The "Decision" column seems to be mostly to blame: Even when it's entirely empty, it causes the header of the "Score W/L/T (Max)" column to be three lines tall, and causes long descriptions (which get truncated with a "...") to be two lines tall.

### PR DESCRIPTION
**Pull request by blackshadowshade**
… and changed Decision to Dismiss for Closed games table

Fixes #1829.

http://jenkins.buttonweavers.com:8080/job/buttonmen-blackshadowshade/982/ (if it passes)

### CODE REVIEW

File: test/src/ui/js/test_Overview.js [hunk 1]
```diff
@@ -399,200 +426,56 @@ test("test_Overview.addInactiveCol", function(assert) {
...
   assert.equal(
     links.length,
     1,
-    'add exactly one link for closed games'
+    'add exactly one link'
   );
   assert.ok(
     links.is('a'),
-    'decision link is correct type for closed games'
+    'decision link is correct type'
```
Comment (cgolubi1) [inline test/src/ui/js/test_Overview.js]: Nit: "dismiss link"

File: test/src/ui/js/test_Overview.js [hunk 2]
```diff
@@ -399,200 +426,56 @@ test("test_Overview.addInactiveCol", function(assert) {
...
   );
   assert.equal(
     links.attr('data-gameid'),
     '500',
-    'decision data-gameid is correct for closed games'
+    'decision data-gameid is correct'
```
Comment (cgolubi1) [inline test/src/ui/js/test_Overview.js]: Nit: "dismiss data-gameid"

File: test/src/ui/js/test_Overview.js [hunk 3]
```diff
@@ -399,200 +426,56 @@ test("test_Overview.addInactiveCol", function(assert) {
...
   );
   assert.equal(
     links.attr('href'),
     '#',
-    'decision href is correct for closed games'
+    'decision href is correct'
```
Comment (cgolubi1) [inline test/src/ui/js/test_Overview.js]: Nit: "dismiss href"

File: test/src/ui/js/test_Overview.js [hunk 4]
```diff
@@ -399,200 +426,56 @@ test("test_Overview.addInactiveCol", function(assert) {
...
-  assert.equal(
-    links.attr('href'),
-    '#',
-    'decision href is correct for new games awaiting opponent'
-  );
-  assert.equal(
-    links.text(),
-    'Cancel',
-    'decision text is correct for new games awaiting opponent'
+    'decision text is correct'
```
Comment (cgolubi1) [inline test/src/ui/js/test_Overview.js]: Nit: "dismiss text"

**Comment by cgolubi1** [comment]
Code looks fine (and i don't care about those nits in any deep way, obviously).  Let me bring it up in virtualbox and take a look.

**Comment by blackshadowshade** [comment]
Nits fixed.

**Comment by cgolubi1** [comment]
Looks good to me.  Merging.

**Pull request merged**